scripts/verify/verify-py.sh

```diff
@@ -21,19 +21,18 @@ create_venv() {
 
 run_pylint() {
     local pylint="${VENV}/bin/pylint"
-   
+
     if ! command -v "${pylint}" &>/dev/null; then
         echo "Installing pylint..."
         create_venv
     fi
-       
-    local pyfiles    
+
+    local pyfiles
     pyfiles=$(find . -type d \( -path ./_output -o -path ./vendor -o -path ./assets -o -path ./etcd/vendor \) -prune -o -name '*.py' -print)
 
-    for f in ${pyfiles} ; do
-        echo "pylint: ${f}"
-        ${pylint} --variable-naming-style=any --score=n "${f}"
-    done
+    # shellcheck disable=SC2086
+    ${pylint} --rcfile="${ROOTDIR}/.pylintrc" ${pyfiles}
```

Comment:
What's the advantage of running the lint on all files at once, rather than processing file-by-file?

Reply:
If we process one file at a time, the job stops at the first file that has an error. If we process them all at once, all of the files are processed without stopping the job, so we get all of the errors at one time. We can also potentially add parallel execution, but I didn't bother with that here.

Reply:
Should we still print the list of files that are checked? I think it might be interesting to see that in the CI to make sure nothing is missed by accident. We could leave the loop printing the names without running the command, and then just run the command all at once.

Reply:
Added